test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- Hey, while you've added the `test.md ` file, you should ensure it's not visible in your repository. Please fix this issue. ensure the `test.md` file is not showing on your repository.